printer_mods/README.md

```diff
@@ -53,6 +51,9 @@ like so:
 | buzzdalf | [v1.8/2.4 Afterburner connector cover for 42STH20-1004AS](./buzzdalf/connector_cover) | v1.8/2.4 Connector cover for 42STH20-1004AS | :x: | :heavy_check_mark: | :heavy_check_mark: |:grey_question:|
 ||[Ender 3 Switchwire Taller Z Mod](./buzzdalf/ender_switchwire_taller_z) | Taller Z Mod | :x: | :x: | :x: | :heavy_check_mark: |
 | chrisrgonzales | [270° Front Panel Hinge](./chrisrgonzales/270_degree_hinge) | Hinges that have 270° of swivel, parametric CAD included | :x: | :x: | :heavy_check_mark: | :x: |
+|claytono|[Fysetc S6 V2 Mount](./claytono/fysetc-s6-v2-mount)| DIN Mount for the Fysetc S6 V2 controller |:x:|:x:|:heavy_check_mark:|:heavy_check_mark:|
```

Comment:
Did you mean to put a checkmark on V2 and Switchwire, instead of V1 and V2?  This mount won't work with a Switchwire, since it doesn't use DIN rails.

Reply:
Nope, I meant to check the V1 column.  Thanks :)